### ISSUE TITLE
support for HTML 5 audio/video elements

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2063]
**Reported by jfauchon on 2012-01-26 18:43**
Hello,

I have a suggestion, can you make support for HTML 5 please. Because, on Internet Explorer and Mozilla Firefox, NVDA not detect for example, audio, video and others html 5 tags.

Thanks

**Comment by bhavyashah** [comment]
@jcsteh @feerrenrut Could the attachment in https://github.com/nvaccess/nvda/issues/2063#issuecomment-155295415 please be restored? Does it demonstrate the inaccessibility reported by the original author on more recent versions of web browsers? If not, I suggest closing as worksforme.

**Comment by feerrenrut** [comment]
I have attached the file to https://github.com/nvaccess/nvda/issues/2063#issuecomment-155295415, @bhavyashah would you be able to test this / investigate further?

**Comment by bhavyashah** [comment]
@feerrenrut Alright. Findings are as follows:
* Firefox version 54.0.1 displays two sliders, Play(/Pause) and Mute buttons, so all works as expected.
* Chrome (latest stable version) also works as with Firefox, with the addition of a Download Media option (unrelated to NVDA and accessibility).
* Internet Explorer 11, NVDA reads, because I believe the browser itself shows - ' This browser is not support audio element.'
I seriously doubt that IE supports these HTML 5 controls inherently and NVDA is unable to read them, because from the message pasted above, it seems to me that the browser, i.e. IE, itself fails to render these HTML 5 elements. Having said that, this seems to be an IE specific issue, and due to its demise, investigating this any further may not be justified. Time to close? @ehollig

**Comment by ehollig** [comment]
Closing, as the HTML5 video element works in Firefox and Chrome, and acording to https://github.com/nvaccess/nvda/issues/2063#issuecomment-155295416, 
"This may not be possible in IE without changes to IE itself."
Closing as Internet Explorer, NeedsExternalFix, cantfix
